cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -623,19 +623,23 @@ module {{bicepName .Name}} 'br/public:avm/res/app/container-app:0.8.0' = {
           {{- range $i, $e := .Frontend.Backends}}
           {
             name: '{{upper .Name}}_BASE_URL'
-            value: 'https://{{.Name}}.${containerAppsEnvironment.outputs.defaultDomain}'
+            value: 'https://${ {{bicepName .Name}}.outputs.name}.${containerAppsEnvironment.outputs.defaultDomain}'
           }
           {{- end}}
           {{- end}}
           {{- if .Backend}}
           {{- range $i, $e := .Backend.Frontends}}
           {
             name: '{{upper .Name}}_BASE_URL'
-            value: 'https://{{.Name}}.${containerAppsEnvironment.outputs.defaultDomain}'
+            value: 'https://{{containerAppName .Name}}.${containerAppsEnvironment.outputs.defaultDomain}'
           }
           {{- end}}
           {{- end}}
           {{- if ne .Port 0}}
+          {
+            name: 'server.port'
```

Comment:
Add server.port Env to explicitly specify exposed port, as petclinic microservice will select random port in config server settings.

Reply:
I remember I did this before, not sure why it's removed in latest code.
Refs: https://github.com/saragluna/azure-dev/pull/9/files